azure-go-aks-helm/cluster.go

```diff
@@ -54,11 +54,6 @@ func buildCluster(ctx *pulumi.Context, cfg Config) (*ClusterInfo, error) {
 	k8sCluster, err := cs.NewManagedCluster(ctx, "cluster",
 		&cs.ManagedClusterArgs{
 			ResourceGroupName: resourceGroup.Name,
-			AddonProfiles: cs.ManagedClusterAddonProfileMap{
```

Comment:
The k8s version bump broke compatibility with this addon so I removed it. It didn't appear to be a significant part of the README.

Reply:
Sounds good.